Automattic/wp-calypso
# Udpate node to 16.7.0

#### Changes proposed in this Pull Request

Update node to 16.7.0. See the [full changelog](https://github.com/nodejs/node/blob/master/doc/changelogs/CHANGELOG_V16.md#16.7.0)

Notable changes:

* From 16.6.0:
    * The V8 engine is updated to version 9.2.230.21. It notably introduces the new `Array.prototype.at` method.
    * Fixes CVE-2021-22930: Use after free on close http2 on stream canceling (High)
* From 16.6.1:
    * Updated npm to 7.20.3
    * Fixed a bug in error handling known to affect at least Webpack and Jest 
* From 16.6.2:
    * Fixes CVE-2021-3672/CVE-2021-22931: Improper handling of untypical characters in domain names
    * Fixes CVE-2021-22930: Use after free on close http2 on stream canceling
    * Fixes CVE-2021-22939: Incomplete validation of rejectUnauthorized parameter
* From 16.7.0:
    * fs: experimental: add recursive cp method

#### Testing instructions

- Run `nvm install && nvm use` and check you can run `yarn`, `yarn start`...
- Verify tests are green
